.github/workflows/main-ci.yml

```diff
@@ -83,15 +83,16 @@ jobs:
                mention: 'here', mention_if: 'failure', commit: true, token: '${{ secrets.SERVICES_GITHUB_TOKEN }}' }
       }]
 
-  bld-eng-macos:
+  bld-eng-darwin:
     needs: [pre-check]
     strategy:
       matrix:
-        platform: [x86_64-macos]
+        platform: [x86_64-macos, arm64-ios, x86_64-ios]
     runs-on: macOS-latest
     steps: [
       { name: 'Checkout', uses: actions/checkout@v3, with: { ref: '${{env.BUILD_BRANCH}}' } },
       { name: 'Install Python', uses: actions/setup-python@v3, with: { python-version: 3.x, architecture: x64 } },
+      { name: 'XCode', uses: maxim-lobanov/setup-xcode@v1, with: { xcode-version: '13.2.1' } },
```

Comment:
Keeping track with the list here:
https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md#xcode

I wonder if we can do something similar when running the extender server.
(to avoid doing the xcode packages at all)